main.js

```diff
@@ -0,0 +1,39 @@
+const submitButton = document.getElementsByClassName('active')[0];
+
+submitButton.addEventListener('click', function () {
+
+const inputPost = document.getElementsByClassName('post')[0].value;
+const inputName = document.getElementsByClassName('name')[0].value;
+
+const newSpan = document.createElement('span');
+newSpan.innerHTML = `${inputPost}, posted by ${inputName}`;
```

Comment:
this is less recommented, use textContent or appendChild as they are safer and more performant.